File: platforms/swift/examples/scene/scene/add_to.swift
import FragmentColor

let mesh = Mesh()
try mesh.addVertex(
    try Vertex.pbr([0.0, 0.5, 0.0]).set("uv0", [0.5, 1.0])
)
let model = Model(mesh, Material.pbr())

let scene = Scene()
scene.addPass(Pass("geometry"))

// Target the pass by name (or pass its index: scene.addTo(0, model)).
try scene.addTo("geometry", model)

File: platforms/web/examples/scene/scene/add_to.js
import { Material, Mesh, Model, Pass, Scene, Vertex } from "fragmentcolor";

const mesh = new Mesh();
mesh.addVertex( Vertex.pbr([0.0, 0.5, 0.0]).set("uv0", [0.5, 1.0]), );
const model = new Model(mesh, Material.pbr());

const scene = new Scene();
scene.addPass(new Pass("geometry"));

// Target the pass by name (or pass its index: scene.addTo(0, model)).
scene.addTo("geometry", model);